tests/shop/test_01.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from src.shop.__init__ import *
+
+
+class Place_order(unittest.TestCase):
```

Comment:
Klasy w Pythonie powinny być nazwane przy użyciu konwencji jaką jest `PascalCase`. Dodatkowo, fajnie by było zaznaczyć jasno, że jest to klasa testowa: `TestPlaceOrder`.